app/mailers/tasks_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class TasksMailer < ApplicationMailer
+  default from: "info@asana.com"
```

Comment:
"from" put to some constant (or in ApplicationMailer)
it will be global for all future emails

do we need this default template_path: 'mailers/tasks' ??